invenio_app_ils/circulation/utils.py

```diff
@@ -12,28 +12,26 @@
 from flask import current_app
 from invenio_pidstore.errors import PersistentIdentifierError
 
-from ..records.api import Document, Item
+from ..records.api import Item
+from ..search.api import ItemSearch
 
 
 def circulation_build_item_ref(item_pid):
     """Build $ref for item."""
     return {
-        "$ref": "{scheme}://{host}{item_resolving_path}{item_pid}".format(
+        "$ref": "{scheme}://{host}/api/resolver/circulation/items/{item_pid}".format(
```

Comment:
why do we change it back to hardcoded url? 😢

Reply:
because we don't want to expose it as it can lead to inconsistencies :)